gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -311,4 +329,31 @@ class BaselineIdea extends AbstractBaselinePlugin {
             module.dependencies.addAll(projectRefs)
         }
     }
+
+    /**
+     * Configures some defaults on the save-actions plugin, but only if it hasn't been configured before.
+     */
+    private static void configureSaveActions(Node rootNode) {
+        GroovyXmlUtils.matchOrCreateChild(rootNode, 'component', [name: 'SaveActionSettings'], [:]) {
+            // Configure defaults if this plugin is configured for the first time only
+            appendNode('option', [name: 'actions']).appendNode('set').with {
+                appendNode('option', [value: 'activate'])
+                appendNode('option', [value: 'noActionIfCompileErrors'])
+                appendNode('option', [value: 'organizeImports'])
+                appendNode('option', [value: 'reformat'])
+            }
+            appendNode('option', [name: 'configurationPath', value: ''])
+            appendNode('inclusions').appendNode('set').with {
+                appendNode('option', [value: 'src/.*\\.java'])
+            }
+        }
+    }
+
+    private static void configureExternalDependencies(Node rootNode) {
+        def externalDependencies = GroovyXmlUtils.matchOrCreateChild(rootNode, 'component', [name: 'ExternalDependencies'])
+        // I kid you not, this is the id for the save actions plugin:
+        // https://github.com/dubreuia/intellij-plugin-save-actions/blob/v1.9.0/src/main/resources/META-INF/plugin.xml#L5
+        // https://plugins.jetbrains.com/plugin/7642-save-actions/
+        GroovyXmlUtils.matchOrCreateChild(externalDependencies, 'plugin', [id: 'com.dubreuia'], ['min-version': '1.9.0'])
```

Comment:
might want to pull out the version so its more visible/easier to upgrade

Reply:
done